src/main/java/dto/ForeignVehicleDto.java

```diff
@@ -0,0 +1,95 @@
+package dto;
+
+import entities.Vehicle;
+import java.io.Serializable;
+import java.util.List;
+
+public class ForeignVehicleDto implements Serializable {
+
+	private long id;
+	private String licensePlate;
+	private String brand;
+	private String type;
+	private String category;
+	private String hardwareSn;
+
+	private String countryCode;
+	private List<JourneyDto> journeys;
+
+	public ForeignVehicleDto(){}
+
+	public ForeignVehicleDto(Vehicle vehicle, List<JourneyDto> journeys){
```

Comment:
Is het niet beter om VehicleDto te gebruiken in plaats van vehicle?

Reply:
Weet niet of dit uitmaakt aangezien in Administratie geen Vehicle class aanwezig is, maar wel een VehicleDto class

Reply:
Ik ben de ForeignVehicleDto in de Administratie backend nu aan het aanpassen zodat deze een VehicleDto heeft ipv Vehicle object.
Voor nu zal ik de PR goedkeuren. Mocht het problemen opleveren kunnen we het altijd nog aanpassen ;)